hs_core/models.py

```diff
@@ -526,6 +526,9 @@ def update(cls, element_id, **kwargs):
             element.metadata.is_dirty = True
             element.metadata.save()
 
+        if(type(kwargs['abstract']) == list):
```

Comment:
Good point... reviewing the hs_restclient code didnt reveal anything obvious at first glance. and if you look at this issue the example code isnt a list